tests/integration/components/bs-form-test.js

```diff
@@ -499,8 +499,8 @@ module('Integration | Component | bs-form', function (hooks) {
 
   test('yielded submit action rejects for expected scenarios', async function (assert) {
     let scenarios = [
-      { validate: sinon.fake.rejects('rejected value') },
-      { onSubmit: sinon.fake.rejects('rejected value') },
+      { validate: sinon.fake.rejects('rejected value'), onSubmit: sinon.fake.resolves() },
```

Comment:
`validate` function was not executed at all before cause `hasValidator` was false. Ember Bootstrap was rejecting cause `onSubmit` was undefined. Explicitly passing both seems to be more clear. Especially as a property from one scenario leaks to the next scenario if not being overwritten.